src/app/builders/components/Table/SelectorHeaderCell.stories.tsx

```diff
@@ -0,0 +1,109 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { useState } from 'react'
+import { SelectorHeaderCell } from './SelectorHeaderCell'
+
+const meta: Meta<typeof SelectorHeaderCell> = {
+  title: 'Koto/Builders/Table/SelectorHeaderCell',
+  component: SelectorHeaderCell,
+  tags: ['autodocs'],
+  parameters: {
+    layout: 'centered',
+  },
+}
+
+export default meta
+
+type Story = StoryObj<typeof SelectorHeaderCell>
+
+export const Default: Story = {
+  render: () => {
+    const [hasSelections, setHasSelections] = useState(false)
+
+    return (
+      <div className="w-[300px] p-4 bg-[#37322F]">
+        <SelectorHeaderCell
+          hasSelections={hasSelections}
+          onClick={() => setHasSelections(prev => !prev)}
+          className="cursor-pointer hover:outline-1 hover:outline-white transition-colors p-2 rounded"
+        />
+        <div className="mt-4 text-sm text-white-600">
+          Current state: {hasSelections ? 'Has selections' : 'No selections'}
+        </div>
+      </div>
+    )
+  },
+}
+
+export const WithSelections: Story = {
+  render: () => (
+    <div className="w-[300px] p-4 bg-[#37322F]">
+      <SelectorHeaderCell hasSelections={true} className="p-2" />
+    </div>
+  ),
+}
+
+export const WithoutSelections: Story = {
+  render: () => (
+    <div className="w-[300px] p-4 bg-[#37322F]">
+      <SelectorHeaderCell hasSelections={false} className="p-2" />
+    </div>
+  ),
+}
+
+export const InTable: Story = {
```

Comment:
I don't see the value of this story.

Reply:
I do. It shows it's usage in the context of a table.